lib/metadata/ip.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metadata
+
+import (
+	"bytes"
+	"fmt"
+	"math"
+	"net"
+	"strconv"
+	"strings"
+)
+
+type IPRange struct {
```

Comment:
You could remove these annotations and move this into /pkg - there's nothing here that's unique to VIC and you don't need the annotations for the fields to be serialized. They will inherit their parent scope.